Header header
geometry_msgs/Point[] object_configuration